Makefile

```diff
@@ -24,6 +24,7 @@ clean-pyc:
 
 lint: build
 	docker-compose run app flake8 --max-line-length 100 .
+	docker-compose run app yamllint repositories.yaml .circleci
```

Comment:
Do we need to add this as a dependency?

Reply:
I didn't know that. Probably not a big deal either way then

Reply:
It's easy to add, so I added it. Will save us if glean ever loses the dep.